go/obscuronode/enclave/README.md

```diff
@@ -0,0 +1 @@
+This package contains code related to the node's enclave component.
```

Comment:
Umm. I think we should elaborate more here. 
Talk a bit how this has to be compiled with ego, started in a separate process, from the main method, etc